cwatch.php

```diff
@@ -970,7 +974,7 @@ private function getMalwareTab($template, $service, $post)
                     'host' => $this->Html->ifSet($post['host']),
                     'port' => $this->Html->ifSet($post['port']),
                     'path' => $this->Html->ifSet($post['path']),
-                    'protocol' => isset($post['ftps']) ? 'FTPS' : 'FTP'
+                    'protocol' => isset($post['sftp']) ? 'SFTP' : 'FTP'
```

Comment:
This should probably be Html::ifSet

Reply:
It is a checkbox, I don't care about the value, only that it is set